src/resources/operation.rs

```diff
@@ -373,6 +377,49 @@ pub struct LiquidityPoolWithdrawOperation {
     pub reserves_received: Vec<AssetAmount>,
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub enum HostFunctionType {
+    HostFunctionTypeHostFunctionTypeInvokeContract,
+    HostFunctionTypeHostFunctionTypeCreateContract,
+    HostFunctionTypeHostFunctionTypeUploadContractWasm,
```

Comment:
I wondered why this was prefixed twice, but then I see it's done also at https://developers.stellar.org/docs/data/horizon/api-reference/resources/operations/object/invoke-host-function. Weird, heh.

Reply:
Would it maybe make sense to remove the prefixes through renaming? Because when referring to this enum in Rust we now would get for example `HostFunctionType::HostFunctionTypeHostFunctionTypeInvokeContract`...